File: backend-ruby/app/models/phan_phong.rb
class PhanPhong < ApplicationRecord
  self.table_name = 'PhanPhong'
  self.primary_key = 'PhanPhongID'
  belongs_to :ky_thi, class_name: "KyThi", foreign_key: "KyThiID"
  belongs_to :phong_thi, class_name: "PhongThi", foreign_key: "PhongThiID"
  belongs_to :nguoi_phan, class_name: "NguoiDung", foreign_key: "NguoiPhanID", optional: true
  belongs_to :dang_ky_thi, class_name: "DangKyThi", foreign_key: "DangKyThiID"
end

File: backend-csharp/Models/PhanPhong.cs
namespace backend_csharp.Models;

public class PhanPhong
{
    public int PhanPhongID { get; set; }
    public int DangKyThiID { get; set; }
    public int KyThiID { get; set; }
    public int PhongThiID { get; set; }
    public int? NguoiPhanID { get; set; }
    public DateTime ThoiDiemPhan { get; set; }
    public KyThi? KyThi { get; set; }
    public PhongThi? PhongThi { get; set; }
    public NguoiDung? NguoiPhan { get; set; }
    public DangKyThi? DangKyThi { get; set; }
}
